### ISSUE TITLE
The training loop shows one extra dimension of loss.

### ISSUE THREAD

**Issue by PravinVedurla** [zzh8829/yolov3-tf2#286]
The number of classes I had for my custom data was just 2, although while training YoloV3 I can see three distinct losses being displayed, do note that all three losses reduce overtime. This wasn't the case when training Yolov3-tiny, then I saw only 2 losses and everything was working as intended. 

There any specific reason for this?

Cmd I used to run the script
`python train.py --batch_size 10 --dataset train.tfrecord --val_dataset test.tfrecord --num_classes 2 --classes logo_class.names --weights_num_classes 80 --epochs 200 --mode eager_tf --transfer darknet --weights ./checkpoints/yolov3.tf`

Note that logo_class.names has only 2lines of plain text, just names of class labels and that's it. 

```
I0606 11:17:58.428167 140495928760192 train.py:156] 1_train_2, 83224.1328125, [9658.231, 21536.658, 52018.652]
I0606 11:17:59.327366 140495928760192 train.py:156] 1_train_3, 74026.609375, [10319.983, 17611.523, 46084.5]
I0606 11:18:00.233876 140495928760192 train.py:156] 1_train_4, 58376.8203125, [9079.438, 10016.953, 39269.805]
I0606 11:18:01.131963 140495928760192 train.py:156] 1_train_5, 46073.8828125, [3175.4614, 9055.111, 33832.676]
I0606 11:18:02.010694 140495928760192 train.py:156] 1_train_6, 33988.234375, [1182.2821, 6306.362, 26488.936]
I0606 11:18:02.875481 140495928760192 train.py:156] 1_train_7, 29651.818359375, [964.7042, 5639.5054, 23036.941]
I0606 11:18:03.732323 140495928760192 train.py:156] 1_train_8, 23937.375, [742.8717, 4019.832, 19163.99]
I0606 11:18:04.621457 140495928760192 train.py:156] 1_train_9, 19713.861328125, [547.42487, 3162.6714, 15993.072]
I0606 11:18:05.569412 140495928760192 train.py:156] 1_train_10, 16408.841796875, [323.97293, 2589.71, 13484.449]
I0606 11:18:06.494202 140495928760192 train.py:156] 1_train_11, 14548.9501953125, [243.84396, 2205.483, 12088.9]
I0606 11:18:07.568682 140495928760192 train.py:156] 1_train_12, 12987.154296875, [193.71529, 1850.4672, 10932.236]
```

**Comment by Morgensol** [comment]
Hi,
This has nothing to do with the amount of classes in your dataset, this is simply due to how yolo tiny is built compared to normal yolo.
The reason why you see 2 in tiny and 3 in normal is because of the anchor masks i believe (there are 2 in tiny and 3 in normal), though im not 100% sure that this is the reason, but i am sure they are 100% independant of your classes.
